### PR TITLE
module cloudwatch_metric_alarm support DatapointsToAlarm config

### PR DESCRIPTION
**Pull request by owanio1992**
##### SUMMARY
module cloudwatch_metric_alarm support DatapointsToAlarm config #2193

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
cloudwatch_metric_alarm

### CODE REVIEW

**Comment by alinabuzachis** [comment]
@owanio1992 Thank you for working on this new feature. Can you please add a changelog fragment https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment and cover this functionality with tests?

File: plugins/modules/cloudwatch_metric_alarm.py [hunk 1]
```diff
@@ -161,6 +161,11 @@
         type: str
         choices: ['ignore', 'evaluate']
         version_added: 9.0.0
+    datapoints_to_alarm:
+        description:
+          - The number of data points that must be breaching to trigger the alarm.
+        required: false
+        type: int
```
Comment (alinabuzachis) [inline plugins/modules/cloudwatch_metric_alarm.py:168]: Please add `version_added: 8.2.0`

**Review by alinabuzachis — APPROVED** [review]

**Comment by alinabuzachis** [comment]
@owanio1992 Can you please rebase? It LGTM. Thank you.

**Comment by owanio1992** [comment]
@alinabuzachis rebase done. thank you

**Review by tremble — APPROVED** [review]
Thanks for taking the time to submit this PR.

One thing for future PRs: Generally the existing tasks in the integration tests shouldn't be modified, and net-new tasks should be added instead.  This helps to ensure that the module changes don't break existing playbooks unless the change has been explicitly marked as a "breaking" change.

**Pull request merged**